### ISSUE TITLE
How's make a client-server-client application ?

### ISSUE THREAD

**Issue by KiPSOFT** [sffc/socketio-file-upload#122]
Hİ,

I need a send file on client-server-client and store file on server. Client to server example is great. which method use for server-client file download or this possible in this library?

**Comment by sffc** [comment]
No, server-to-client is not currently available with this library.  I would consider it out of scope for now, but perhaps it could be included as part of the work for #118.
